projects/step-frontend/src/lib/modules/plan/injectables/pure-plan-edit-api.service.ts

```diff
@@ -54,8 +56,18 @@ export class PurePlanEditApiService implements PlanEditorApiService {
     return this._planApi.restorePlanVersion(id, versionId);
   }
 
-  savePlan(plan: Plan): Observable<{ id: string; plan: Plan }> {
-    return this._planApi.savePlan(plan).pipe(map((response) => ({ id: response.id!, plan: response })));
+  savePlan(plan: Plan): Observable<{ id: string; plan: Plan; forceRefresh?: boolean }> {
+    return this._planApi.savePlan(plan).pipe(
+      map((response) => ({ id: response.id!, plan: response })),
+      catchError((e: any) => {
```

Comment:
we dont seem to need to make use of the error, so it can be removed

Reply:
Thanks!